src/api/services/PriceTickerService.ts

```diff
@@ -92,14 +114,37 @@ export class PriceTickerService {
         await this.priceTickerRepo.destroy(id);
     }
 
```

Comment:
search method jsdoc should explain what the currency param should be exactly. should it be bitcoin or the symbol btc? Right now I cant tell that by looking at this code.

it should be symbol.
also, why is the return value a Collection when the search parameter implies that we are searching just for one currency?
search method should be renamed to getOneBySymbol()

Reply:
done